composer.lock

```diff
@@ -1196,6 +1196,132 @@
             "time": "2015-11-10 12:26:42"
         },
         {
+            "name": "league/tactician",
```

Comment:
New dependencies have been introduced but the composer.*json* file is the same.

![fucky](http://i.imgur.com/IgU4Y.gif)